Dockerfile

```diff
@@ -1,11 +1,9 @@
 FROM php:8.0-cli-alpine3.13
 
-RUN apk update && \
-    apk add --no-cache \
-        libzip-dev \
-        openssl-dev && \
-    docker-php-ext-install -j$(nproc) \
-        zip
+RUN apk update \
+    && apk add --no-cache $PHPIZE_DEPS libzip-dev openssl-dev \
```

Comment:
Where is $PHPIZE_DEPS defined?

Reply:
This is an env var related to the Alpine distro, this resolves to the required packages to build PHP extensions.